### ISSUE TITLE
Make something to create mouse macros!

### ISSUE THREAD

**Issue by JamBot3000** [themonkeyhacker/Hacktober-Challenges#6]
This one isn't too hard at all! Use a library such as "pynput" to record mouse movements and then be able to repeat them :)

Here are the docs [https://pythonhosted.org/pynput/mouse.html](url)

**Comment by bachlis** [comment]
hey, I find this fun and interesting.. may I work on it?

**Comment by JamBot3000** [comment]
@bachlis of course! :) I don't assign issues to anyone, as I like multiple people to be able to work on the challenges even after someone has done them. Feel free to work on this though :)

**Comment by bachlis** [comment]
hey @JamBot3000 I have submitted my PR, waiting for approval..

**Comment by JamBot3000** [comment]
Epic, I get home around 18:00-19:00 GMT, will review when im home :) Happy hacktoberfest!

**Comment by bachlis** [comment]
hey @JamBot3000 all good with the pr? Any suggested improvements? 🙂
